stdlib/asyncio/unix_events.pyi

```diff
@@ -59,7 +62,19 @@ if sys.version_info < (3, 14):
 
 if sys.platform != "win32":
     if sys.version_info >= (3, 14):
-        __all__ = ("SelectorEventLoop", "DefaultEventLoopPolicy")
+        __all__ = ("SelectorEventLoop", "DefaultEventLoopPolicy", "EventLoop")
```

Comment:
I have no idea why `__all__` is down here, but you could also optionally move it to the top of the file.

Reply:
Addressed: https://github.com/python/typeshed/pull/12327/commits/bee3ed25dd11701b855c0f314936097f01b2869c